static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -192,14 +192,16 @@ function useReplayData({
     };
   }, [orgSlug, replayId, projectSlug, queryClient]);
 
+  const attachments = attachmentPages.flat(2);
```

Comment:
is there a reason behind this one? My intuition is that this `flat(2)` call will always return a new array, and will defeat the useMemo call below

Reply:
thanks forgot to revert these changes